analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Invocation.cs

```diff
@@ -196,6 +199,25 @@ private static ProgramState[] ProcessEquals(SymbolicContext context, IInvocation
         return context.State.ToArray();
     }
 
+    private static ProgramState[] ProcessNullableGetValueOrDefault(SymbolicContext context, IInvocationOperationWrapper invocation)
+    {
+        var instanceValue = context.State[invocation.Instance];
+        if (instanceValue?.HasConstraint(ObjectConstraint.Null) is true)
```

Comment:
the control flow with the `instanceValue?.` and `else if (instanceValue is not null)` is hard to understand. It would be better to do it like so:
```suggestion
        if (context.State[invocation.Instance] is {} instanceValue)
        {
            // This might also be a ternary:
            if (instanceValue.HasConstraint(ObjectConstraint.Null))
            {
                return ;
            }
            else
            {
                return ;
            }
        }
        return context.State.ToArray();
```